configs/antv.json

```diff
@@ -38,44 +38,31 @@
       ]
     },
     {
-      "url": "https://antv.alipay.com/zh",
+      "url": "https://antv.vision/zh.*",
       "tags": [
         "zh"
       ]
     },
     {
-      "url": "https://antv.alipay.com/zh/.*",
+      "url": "https://*.antv.vision/zh.*",
       "tags": [
         "zh"
       ]
     },
     {
-      "url": "https://antv.alipay.com/.*/zh/.*",
-      "tags": [
-        "zh"
-      ]
-    },
-    {
-      "url": "https://antv.alipay.com/en",
-      "tags": [
-        "en"
-      ]
-    },
-    {
-      "url": "https://antv.alipay.com/en/.*",
+      "url": "https://antv.vision/en.*",
```

Comment:
```suggestion
      "url": "https://antv.vision/en",
```

Reply:
I need to cover `https://antv.vision/en` and `https://antv.vision/en/xxx` at the same time.